src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -385,15 +385,18 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
     put_cc_response = create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait)
     put_cc_response = LongRunningOperation(cmd.cli_ctx)(put_cc_response)
     print("Azure resource provisioning has finished.")
-    # Checking if custom locations rp is registered and fetching oid if it is registered
-    enable_custom_locations, custom_locations_oid = check_cl_registration_and_get_oid(cmd, cl_oid, subscription_id)
+
+    # Checking if custom locations rp is registered and verify passed-in cl_oid.
+    test_custom_location_requirements(cmd, cl_oid, subscription_id)
 
     print("Starting to install Azure arc agents on the Kubernetes cluster.")
+
     # Install azure-arc agents
     utils.helm_install_release(cmd.cli_ctx.cloud.endpoints.resource_manager, chart_path, subscription_id, kubernetes_distro, kubernetes_infra, resource_group_name, cluster_name,
```

Comment:
this is insane, are these parameters ordered? how could one keep this straight? Didn't you miss passing one bool param here from before?

Reply:
Can you clarify about the missing Boolean param? Don't remember what that refers to. Yea, this function has a lot of parameters. Might be a way to simplify it, but I will leave that for another PR.

Reply:
Right here in this "utils.helm_install_release" function call, previously the parameters were passed:

    ..., disable_auto_upgrade, enable_custom_locations, custom_locations_oid, helm_client_location, ...

And after the change:

    ..., disable_auto_upgrade, cl_oid, helm_client_location, ...
    
Previously there was another parameter "enable_custom_locations" which I assume is a boolean flag? Hard to follow as all these appear as an array of parameters to this function.